### PR TITLE
ISSUE_TEMPLATE: add incubating program issue template

### PR DESCRIPTION
**Pull request by winkyao**
add an issue template for TiDB incubating program

### CODE REVIEW

File: .github/ISSUE_TEMPLATE/incubating-program.md [hunk 1]
```diff
@@ -0,0 +1,11 @@
+---
+name: "Incubating Program"
+about: Incubating program are activities that the TiDB community uses to incubate specific new features or projects. Participants can submit RFCs to this repo first. If the RFC is approved by most PMC members, an incubation project group will be established, which will involve community contributors (including PingCAP members) and a working group will be established in Community to promote the project.
+label: ""                                                                                                                                                                                                                                     ---
```
Comment (zz-jason) [inline .github/ISSUE_TEMPLATE/incubating-program.md]: please remove the training spaces.

File: .github/ISSUE_TEMPLATE/incubating-program.md [hunk 2]
```diff
@@ -0,0 +1,13 @@
...
+about: Incubating program are activities that the TiDB community uses to incubate specific new features or projects. Participants can submit RFCs to this repo first. If the RFC is approved by most PMC members, an incubation project group will be established, which will involve community contributors (including PingCAP members) and a working group will be established in Community to promote the project.
+label: ""
+---
+
+## Incubating Program
+
+### Where is your RFC/Proposal address?
+<!-- A link to the RFC/Proposal. -->
+
+### Describe the feature or project you want to incubate:
```
Comment (zz-jason) [inline .github/ISSUE_TEMPLATE/incubating-program.md]: Should this part come before the RFC link/address?

File: .github/ISSUE_TEMPLATE/incubating-program.md [hunk 3]
```diff
@@ -0,0 +1,14 @@
+---
+name: "Incubating Program"
+about: Incubating program are activities that the TiDB community uses to incubate specific new features or projects. Participants can submit RFCs to this repo first. If the RFC is approved by most PMC members, an incubation project group will be established, which will involve community contributors (including PingCAP members) and a working group will be established in Community to promote the project.
```
Comment (dcalvin) [inline .github/ISSUE_TEMPLATE/incubating-program.md]:
```suggestion
about: Incubating programs are activities that the TiDB community uses to incubate specific features or projects. Participants can submit proposals as Request for Change (RFC) documents to this repo first. If the RFC is approved by the majority of PMC members, an incubation project group that involves community contributors will be established. A corresponding working group will be established to promote the project.
```

Comment (dcalvin) [reply .github/ISSUE_TEMPLATE/incubating-program.md]: incubation project group and working group - are they the same?

File: .github/ISSUE_TEMPLATE/incubating-program.md [hunk 4]
```diff
@@ -0,0 +1,14 @@
...
+label: ""
+---
+
+## Incubating Program
+
+### Describe the feature or project you want to incubate:
+<!-- A description of what you want to happen. -->
+
+### Where is your RFC/Proposal address?
```
Comment (dcalvin) [inline .github/ISSUE_TEMPLATE/incubating-program.md]:
```suggestion
### Where is your RFC/Proposal?
```

File: .github/ISSUE_TEMPLATE/incubating-program.md [hunk 5]
```diff
@@ -0,0 +1,14 @@
+---
+name: "Incubating Program"
+about: Incubating programs are activities that the TiDB community uses to incubate specific features or projects. Participants can submit proposals as Request for Change (RFC) documents to this repo first. If the RFC is approved by the majority of PMC members, an incubation project group that involves community contributors will be established. A corresponding working group will be established to promote the project.
```
Comment (dcalvin) [inline .github/ISSUE_TEMPLATE/incubating-program.md]:
```suggestion
about: Incubating programs are projects that the TiDB community uses to incubate specific features or projects. Participants can submit proposals as Request for Change (RFC) documents to this repo first. If the RFC is approved by the majority of PMC members, an incubation project group that involves community contributors will be established. A corresponding working group will be established to steer the project.
```

File: .github/ISSUE_TEMPLATE/incubating-program.md [hunk 6]
```diff
@@ -0,0 +1,14 @@
...
+label: ""
+---
+
+## Incubating Program
+
+### Describe the feature or project you want to incubate:
+<!-- A description of what you want to happen. -->
+
+### Where is your RFC/Proposal?
```
Comment (dcalvin) [inline .github/ISSUE_TEMPLATE/incubating-program.md]:
```suggestion
### Your RFC/Proposal?
```

**Pull request merged**